python/pyspark/sql/functions/builtin.py

```diff
@@ -16840,6 +16840,54 @@ def map_concat(
     return _invoke_function_over_seq_of_columns("map_concat", cols)  # type: ignore[arg-type]
 
 
+@_try_remote_functions
+def map_sort(col: "ColumnOrName", asc: bool = True) -> Column:
```

Comment:
let's also put it in the docs at `python/docs/source/reference/pyspark.sql/functions.rst`.